src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup Label="Targets required for unit tests to run">
-    <TargetFrameworkNetDesktop461>net461</TargetFrameworkNetDesktop461>
+    <TargetFrameworkNetDesktop461>net462</TargetFrameworkNetDesktop461>
```

Comment:
Update the element name for consistency - `TargetFrameworkNetDesktop462`

Reply:
that would also need an update to `platform_and_feature_flags.props`